include/ceed-hash.h

```diff
@@ -20,7 +20,7 @@
 #define _ceed_hash_h
 
 #include <ceed.h>
-#include <khash.h>
+#include <ceed-khash.h>
```

Comment:
Renaming the file isn't enough to enable a user to include their own `khash.h` separately. Could you update the header guard in `ceed-khash.h`?

Reply:
updated